RockWeb/Blocks/Crm/PersonDetail/EditGroup.ascx.cs

```diff
@@ -1132,25 +1126,14 @@ protected void btnSave_Click( object sender, EventArgs e )
                                 return;
                             }
 
-                            if ( _isFamilyGroupType )
-                            {
-                                if ( person.RecordStatusValueId != recordStatusValueID )
```

Comment:
Got it, thanks for the explanation, I'll put these lines back as they were. To the best of my recollection I thought I had determined that these lines were either creating bad data or would never get hit, but I just confirmed that they do get hit in your scenario.